apis/clusters/v1beta1/postgresql_types.go

```diff
@@ -88,7 +88,13 @@ type PgSpec struct {
 
 // PgStatus defines the observed state of PostgreSQL
 type PgStatus struct {
-	ClusterStatus `json:",inline"`
+	ClusterStatus  `json:",inline"`
+	DefaultUserRef `json:",inline"`
+}
+
+type DefaultUserRef struct {
+	DefaultUserName      string `json:"defaultUserName,omitempty"`
+	DefaultUserNamespace string `json:"defaultUserNamespace,omitempty"`
 }
```

Comment:
`DefaultUserRef` -} `DefaultUserSercretRef`

The same story for fields inside DefaultUserRef. Also, i think inlining  DefaultUserRef is unnecessary

Reply:
fixed